ubuntu-installer-classic-23.04-amd64.json

```diff
@@ -0,0 +1,41 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "model": "ubuntu-installer-classic-23.04-amd64",
+    "architecture": "amd64",
+    "timestamp": "2023-03-19T12:00:00.0Z",
+    "base": "core22",
+    "grade": "signed",
+    "classic": "true",
+    "distribution": "ubuntu",
+    "snaps": [
+        {
+            "name": "core22",
+            "type": "base",
+            "default-channel": "latest/stable",
+            "id": "amcUKQILKXHHTlmSa7NMdnXSx02dNeeT"
+        },
+        {
+            "name": "snapd",
+            "type": "snapd",
+            "default-channel": "latest/stable",
+            "id": "PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4"
+        },
+        {
+            "name": "subiquity",
+            "type": "app",
+            "classic": "true",
+            "default-channel": "latest/stable/canary",
```

Comment:
wait a minute, we should be seeding a branch of ubuntu-desktop-installer here? or is the canary installer going to be like that other thing which has uses the text ui?

Reply:
if it is subiquity can it track latest/stable/canary-23.04 for consistency with other things please?